contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -3243,6 +6807,8 @@ AS 'babelfishpg_tsql', 'update_user_catalog_for_guest';
  
 CALL sys.babelfish_update_user_catalog_for_guest();
 
+-- comment for the review: Check https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/708 to see if depcrecated
+-- view can cause any collation conflict to cx.
 ALTER VIEW sys.sp_sproc_columns_view RENAME TO sp_sproc_columns_view_deprecated_in_2_3_0;
```

Comment:
This view hasn't been adjusted for collation and should be (both the original object and the upgrade script) . The previous version is of this object is indeed deprecated as the new version has correct datatypes. Are there any specific questions regarding this view?

NOTE: There are some comparisons between the `ss` table from the inner query. This `ss` table consist of postgres tables with the pg collation.